src/frontend/components/SelectContent/index.tsx

```diff
@@ -389,7 +395,17 @@ export const SelectContent = ({
             selectContent={selectContent}
           />
         </Tab>
+
+        <Suspense fallback={<Loader />}>
+          {tabs.map((LazyComponent, index) => (
+            <LazyComponent key={index} selectContent={selectContent} />
+          ))}
+        </Suspense>
       </Tabs>
     </Box>
   );
 };
+
+export interface SelectContentTabProps {
```

Comment:
I don't understand why you defined this interface and why here ? Should be moved in the tests ?

Reply:
I thought it would be useful to export this for defining a SelectContentTab in an app.
See src/frontend/apps/bbb/SelectContentTab/index.tsx in #1107 for an example.

Reply:
:+1: We will see its usage later then.